src/pages/workspace/WorkspacesListPage.js

```diff
@@ -28,6 +28,7 @@ import * as ReimbursementAccountProps from '../ReimbursementAccount/reimbursemen
 import * as ReportUtils from '../../libs/ReportUtils';
 import * as CurrencyUtils from '../../libs/CurrencyUtils';
 import withPolicyAndFullscreenLoading from './withPolicyAndFullscreenLoading';
+import withWindowDimensions from '../../components/withWindowDimensions';
```

Comment:
Minor feedback for next time, we have hooks to replace many of our HOCs, and we should strive to use them going forward in functional components

- `withWindowDimensions` => `useWindowDimensions`
- `withLocalize` => `useLocalize`
- `withNetwork` => `useNetwork`

etc...

whenever refactoring a class component or adding new HOCs, please check `src/hooks` first. Thanks! 😄